api/v1alpha4/cluster_webhook_test.go

```diff
@@ -46,58 +48,282 @@ func TestClusterDefault(t *testing.T) {
 	g.Expect(c.Spec.ControlPlaneRef.Namespace).To(Equal(c.Namespace))
 }
 
-func TestClusterValidation(t *testing.T) {
-	valid := &Cluster{
+func TestClusterDefaultTopologyVersion(t *testing.T) {
+	// NOTE: ClusterManagedTopologies feature flag is disabled by default, thus preventing to set Cluster.Topologies.
+	// Enabling the feature flag temporarily for this test.
+	defer utilfeature.SetFeatureGateDuringTest(t, feature.Gates, feature.ClusterManagedTopologies, true)()
```

Comment:
Just out of curiosity. Isn't that like changing a global singleton. Why doesn't this affect other tests running in parallel? (or do we not run them in parallel?)

Reply:
This was the recommended way in api-machinery (which provides the foundation for our feature flags).
I did not investigated further, but TBH I don't think we are running test in parallel

Reply:
Thx for providing context. I think if it would be real problem it would be really visible in our test runs so we should be fine in any case.